Podfile

```diff
@@ -222,8 +222,8 @@ abstract_target 'Apps' do
 
   pod 'Gridicons', '~> 1.1.0'
 
-  pod 'WordPressAuthenticator', '~> 3.2', '>= 3.2.1'
-  # pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => 'fix/showcancel'
+  pod 'WordPressAuthenticator', '~> 3.2', '>= 3.2.2-beta.1'
+  # pod 'WordPressAuthenticator', git: 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', branch: 'fix/prologue-nav-bar'
```

Comment:
Just a nit, but should we consider reverting this back to the previous format now that it's not in use anymore (the colons were shuffled around)?

Reply:
@twstokes I think all those commented pods should eventually be updated to use the new hash syntax. Otherwise, `wpmobilebot` may complain with this error https://github.com/wordpress-mobile/WordPress-iOS/pull/19461#discussion_r994679671

Reply:
Oh, apologies! This makes sense.